hack/verify-spelling.sh

```diff
@@ -38,7 +38,7 @@ URL="https://github.com/client9/misspell"
 echo "Cloning ${URL} in ${TMP_DIR}..."
 git clone --quiet --depth=1 "${URL}" "${TMP_DIR}"
 pushd "${TMP_DIR}" > /dev/null
-go mod init
+go mod init misspell
```

Comment:
are we aligning with the go version of k/k

Reply:
the version of go that is used in CI is the version in kubekins (test-infra) and that matches the version of k/k - 1.13.4.

Reply:
sgtm

Reply:
/hold cancel